jwcrypto/jwa.py

```diff
@@ -1053,6 +1053,42 @@ class _A256Gcm(_AesGcm, JWAAlgorithm):
     algorithm_use = 'enc'
 
 
+class _BP256R1(_RawEC, JWAAlgorithm):
+
+    name = "BP256R1"
+    description = "ECDSA using Brainpool256R1 curve and SHA-256"
```

Comment:
You should add the "unregistered" description bit here too (and in the other 2 algorithms).